docs/reference/commandline/service_create.md

```diff
@@ -51,10 +51,10 @@ Options:
       --restart-max-attempts value       Maximum number of restarts before giving up (default none)
       --restart-window value             Window used to evaluate the restart policy (default none)
       --stop-grace-period value          Time to wait before force killing a container (default none)
-      --update-delay duration            Delay between updates
+      --update-delay duration            Delay between updates (Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h') (default 0s)
```

Comment:
Sorry for the nitpick 😿 This part of the docs should be just a dump of what you'd get by running `--help` on the CLI.
So according to the changes in `cli/command/service/opts.go`, it should be

```
Delay between updates (ns|us|ms|s|m|h) (default 0s)
```
And the same to the rest of the `/docs/reference` files.

```